src/pages/experiments/[experimentId]/data-processing/data-integration/components/CalculationConfig.jsx

```diff
@@ -182,20 +170,37 @@ const CalculationConfig = () => {
           </Form.Item>
           <Form.Item label='Method:'>
             <Select
-              value={settings.dimensionalityReduction.method}
+              value={dimensionalityReduction.method}
               onChange={(val) => updateSettings({ dimensionalityReduction: { method: val } })}
             >
               <Option value='rpca'>Reciprocal PCA (RPCA)</Option>
               <Option value='cca'>Cannonical Correlation Analysis (CCA)</Option>
             </Select>
           </Form.Item>
           <Form.Item>
-            <Button size='small' type='primary' htmlType='submit' disabled={!changesOutstanding} onClick={() => { setSavedSettings(settings); }}>Apply</Button>
+            <Row>
+              <Col span={6}>
+                <Button
+                  size='small'
+                  type='primary'
+                  htmlType='submit'
+                  disabled={!changesOutstanding}
+                  onClick={applyDataIntegrationSettings}
+                >
+                  Apply
```

Comment:
It seems a bit weird how the apply button is disabled initially. I understand why it is disabled initially -- because there are no changes outstanding, however it does take a few brain-cycles to realise that. My concern is that users who will have much less context about this system won't realise why this button is disabled initially and they would think that they can't change the settings. To mitigate this, I would add a tooltip that appears on hover over this button when there are no changes outstanding that states why the button is disabled and that the user needs to change the configuration settings to enable the button. This issue also appears in Configure Embedding, we can add that tooltip there as well. @vickymorrison what do you think? Is there anything better that we can do here?

Reply:
Iva's suggestion is a good immediate solution.
It raises the bigger question of whether users will realise that Data Integration has already been performed. I know that Adam's wireframes for the filters have a toggle at the top with 'default' and 'manual' options which make it clear that default settings have already been applied to the data. Iva and I will discuss whether to apply this to Data Integration and Configure Embedding too.
But for now, adding the tooltip is a good way forward.

Reply:
Ok. I've added the tooltip both in Data Integration and Configure Embedding